app/views/benefits/edit.html.slim

```diff
@@ -18,4 +18,8 @@
 
     = render "form", form:
 
-= render "shared/disregarded_benefits_sidebar"
+- if FeatureFlags.enabled?(:controlled)
```

Comment:
I don't think you need to check the feature flag here: If `:controlled` is _not_ enabled then @estimate.level_of_help will never be equal to `"controlled"`, so your `level_of_help` assignment below still works. So lines 21, 24 and 25 can be deleted, because lines 22 and 23 cover all cases.

Reply:
I ended up wrapping these sidebars in the feature flag check because it was blowing up some tests without it. turns out @estimate was not available in edit/update hence only some tests breaking.